project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/my_groups/my_groups.inc

```diff
@@ -70,7 +70,7 @@ function c4m_features_homepage_get_user_groups($account = NULL, $allowed_types =
   $groups = c4m_user_og_get_groups($account);
 
   if (empty($groups)) {
-    return FALSE;
+    return array();
```

Comment:
Already did that change: https://github.com/capacity4dev/capacity4more/pull/1139/files#diff-1056a588a4b868e01a197b176b661a81R71